content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

```diff
@@ -2,40 +2,122 @@
+To view (and edit) locations, navigate to :menuselection:`Inventory app --> Configuration -->
+Locations`. This reveals a :menuselection:`Locations` page containing every location currently
+created and listed in the warehouse.
 
-Then, click on a location to open the location settings. Next, click on :guilabel:`Edit`. In the
-:guilabel:`Inventory Frequency (Days)` field, set the number of days. For example, a location that
-needs an inventory count every 30 days would set the :guilabel:`Inventory Frequency (Days)` value
-to `30`. Once the value is entered, click :guilabel:`Save` to apply the setting to the location.
-Now, once an inventory adjustment is applied to this location, the next scheduled count date will
-be automatically set based on the number of days in the :guilabel:`Inventory Frequency (Days)`
-setting.
+From this page, click into a location to reveal the settings and configuration page for that
+location. Click :guilabel:`Edit` to edit the location settings.
 
-.. image:: cycle_counts/inventory-frequency.png
+Under the :guilabel:`Cyclic Inventory` section, locate the :guilabel:`Inventory Frequency (Days)`
+field, which should be set to `0` (if this location has not been edited previously). In this field,
+change the value to whichever number of days is desired.
+
+.. image:: cycle_counts/cycle-counts-location-frequency.png
    :align: center
-   :alt: Edit a location to change the inventory frequency.
+   :alt: Location frequency setting on location.
+
+.. example::
+   A location that needs an inventory count every 30 days should have the :guilabel:`Inventory
+   Frequency (Days)` value set to `30`.
+
+Once the frequency has been changed to the desired number of days, click :guilabel:`Save` to save
+changes. Now, once an inventory adjustment is applied to this location, the next scheduled count
+date will be automatically set based on the value entered into the :guilabel:`Inventory Frequency
+(Days)` field.
+
+Count inventory by location
+===========================
+
+To perform a cycle count for a specific location in the warehouse, navigate to
+:menuselection:`Inventory app --> Operations --> Inventory Adjustments`. This reveals an
+:guilabel:`Inventory Adjustments` page containing all products currently in stock, with each product
+listed on its own line.
+
+From this page, the :guilabel:`Filters` and :guilabel:`Group By` buttons (at the top of the page,
+under the :guilabel:`search bar`), can be used to select specific locations and perform inventory
+counts.
+
+.. image:: cycle_counts/cycle-counts-inventory-adjustments-page.png
+   :align: center
+   :alt: Inventory adjustments page.
+
+To select a specific location and view all products within that location, click :guilabel:`Group
+By`, then click :guilabel:`Add Custom Group` to reveal a new drop-down menu to the right. Click this
+menu to reveal the drop-down, and select :guilabel:`Location`. Then, click :guilabel:`Apply`. The
```

Comment:
Click :guilabel:`Location`  from the drop-down menu, then click :guilabel:`Apply`.

Reply:
The drop-down was already revealed by clicking Add Custom Group, so I just wanted to clear that up and streamline this.